src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -270,17 +201,17 @@ $(XamlMetaDataProviderPch)
     <!--Insert Midl /references to Platform WinMDs, library reference WinMDs, and direct reference WinMDs-->
     <Target Name="CppWinRTSetMidlReferences"
             BeforeTargets="Midl"
-            DependsOnTargets="GetCppWinRTPlatformWinMDReferences;CppWinRTAddXamlMetaDataProviderIdl;GetCppWinRTStaticWinMDReferences;GetCppWinRTDynamicWinMDReferences;GetCppWinRTDirectWinMDReferences;$(CppWinRTSet_MidlReferencesDependsOn)">
+            DependsOnTargets="GetCppWinRTPlatformWinMDReferences;CppWinRTAddXamlMetaDataProviderIdl;GetCppWinRTProjectWinMDReferences;GetCppWinRTDirectWinMDReferences;$(CppWinRTSet_MidlReferencesDependsOn)">
```

Comment:
I don't see where GetCppWinRTProjectWinMDReferences is defined

Reply:
or ResolveProjectReferences hooked for that matter

Reply:
Looks like it got removed in the second iteration. I'll add it back.